pkg/startup/v11/data/master/etc/aro-admission-controller/aro-admission-controller.yaml

```diff
@@ -0,0 +1,5 @@
+whitelist:
+  - {{ .Config.Images.ControlPlane | quote }}
```

Comment:
1) are these going to be regexes?  If so need ^ $ and a quotemeta
2) can the admission controller detect pods that are going to be scheduled to masters (via their nodeselector?) and ignore them, such that they need not be listed here?